probe/core/pom.xml

```diff
@@ -0,0 +1,211 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>weld-probe-parent</artifactId>
+        <groupId>org.jboss.weld.probe</groupId>
+        <version>2.3.0.Beta1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>weld-probe-core</artifactId>
+    <name>Weld Probe Core</name>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>javax.enterprise</groupId>
+            <artifactId>cdi-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.weld</groupId>
+            <artifactId>weld-core-impl</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.spec.javax.servlet</groupId>
+            <artifactId>jboss-servlet-api_3.0_spec</artifactId>
```

Comment:
The 2.3 branch has been updated recently to use `jboss-servlet-api_3.1_spec` and other more up to date API artifacts.  See https://github.com/weld/core/commit/690e77ba016433acfe77df9356435f92618c45df Update and rebase this PR accordingly.